build_tools/cmake/iree_native_test.cmake

```diff
@@ -4,7 +4,31 @@
 # See https://llvm.org/LICENSE.txt for license information.
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
-include(CMakeParseArguments)
+# iree_parse_input_file_arg
+#
+# A helper function to split the flag and the file path from
+# `TEST_INPUT_FILE_ARGS`. it is for `iree_run_module_test` as `iree-run-module`
+# can have multiple input files while are passed through different flags.
+# The file path is then handled in `iree_native_test` for portability.
+function(iree_parse_input_file_arg OUT_FLAG OUT_FILE INPUT_ARG)
```

Comment:
Oof this is a bit icky. We're taking a string flag and parsing it back out into structured data. How about doing the reverse and passing the flags with placeholders (index of the file arg, perhaps?) and then filling in the appropriate file paths. Either something like

```CMake
ARGS
  "--input={{0}}"
TEST_INPUT_FILES
  "foo.npy"
```

or

```CMake
ARGS
  "--input={{foo.npy}}"
```

(syntax just a strawman). The latter would match Bazel's `$location`

Reply:
Done. PTAL.